projects/downloads.py

```diff
@@ -0,0 +1,158 @@
+from blueprintapi.oscal.oscal import Metadata
+from blueprintapi.oscal.ssp import (
+    BackMatter,
+    ByComponent,
+    Component,
+    ControlImplementation,
+    Impact,
+    ImplementedRequirement,
+    ImportProfile,
+    InformationType,
+    Model,
+    NetworkDiagram,
+    Party,
+    Resource,
+    Role,
+    SecurityImpactLevel,
+    SystemCharacteristics,
+    SystemImplementation,
+    SystemInformation,
+    SystemSecurityPlan,
+    SystemStatus,
+)
+from components.componentio import ComponentTools
+from projects.models import Project
+
+
+class OscalSSP: # pylint: disable=too-many-instance-attributes
+    metadata = None
+    sec_impact_level = None
+    import_profile = None
+    information_type = None
+    system_information = None
+    system_characteristics = None
+    component_ref = {}
+    system_implementation = None
+    control_implementations = None
+    back_matter = None
```

Comment:
Should all of these be class attributes? Instances of this class will share these values, so was that intentional?